sql/mob_skill_lists.sql

```diff
@@ -3334,4 +3334,311 @@ INSERT INTO `mob_skill_lists` VALUES ('Durinn',929,2114); -- Hellclap
 INSERT INTO `mob_skill_lists` VALUES ('Durinn',929,2115); -- Cackle
 INSERT INTO `mob_skill_lists` VALUES ('Durinn',929,2116); -- Necrobane
 INSERT INTO `mob_skill_lists` VALUES ('Durinn',929,2117); -- Necropurge
-INSERT INTO `mob_skill_lists` VALUES ('Durinn',929,2118); -- Bilgestorm
\ No newline at end of file
+INSERT INTO `mob_skill_lists` VALUES ('Durinn',929,2118); -- Bilgestorm
+
+INSERT INTO `mob_skill_lists` VALUES ('Amarok',930,1785); -- Lava_Spit
+INSERT INTO `mob_skill_lists` VALUES ('Amarok',930,1786); -- Sulfurous_Breath
+INSERT INTO `mob_skill_lists` VALUES ('Amarok',930,1787); -- Scorching_Lash
+INSERT INTO `mob_skill_lists` VALUES ('Amarok',930,1788); -- Ululation
+INSERT INTO `mob_skill_lists` VALUES ('Amarok',930,1789); -- Magma_Hoplon
+
+INSERT INTO `mob_skill_lists` VALUES ('Ansherekh',931,270); -- Roar
+INSERT INTO `mob_skill_lists` VALUES ('Ansherekh',931,271); -- Razor_Fang
+INSERT INTO `mob_skill_lists` VALUES ('Ansherekh',931,273); -- Claw_Cyclone
+
+INSERT INTO `mob_skill_lists` VALUES ('Battlerigged_Chariot',932,2054); -- Diffusion_ray
+INSERT INTO `mob_skill_lists` VALUES ('Battlerigged_Chariot',932,2055); -- Intertia_stream
+
+INSERT INTO `mob_skill_lists` VALUES ('Bennu',933,399); -- Scratch
+INSERT INTO `mob_skill_lists` VALUES ('Bennu',933,400); -- Triple_Attack
+INSERT INTO `mob_skill_lists` VALUES ('Bennu',933,401); -- Gliding_Spike
+INSERT INTO `mob_skill_lists` VALUES ('Bennu',933,402); -- Feather_Barrier
+INSERT INTO `mob_skill_lists` VALUES ('Bennu',933,922); -- Blind_Vortex
+INSERT INTO `mob_skill_lists` VALUES ('Bennu',933,923); -- Giga_Scream
+-- INSERT INTO `mob_skill_lists` VALUES ('Bennu',933,2703); -- Dread_Wind
+
+INSERT INTO `mob_skill_lists` VALUES ('Bugul_Noz',934,2210); -- Petal_Pirouette
+
+INSERT INTO `mob_skill_lists` VALUES ('Bugul_Noz_post_50',935,2210); -- Petal_Pirouette 
+-- INSERT INTO `mob_skill_lists` VALUES ('Bugul_Noz_post_50',935,2601); -- Petalback_Spin
```

Comment:
its a really small bit in the skillcheck instead of checking if it shoudl change lists every cycle of onmobfight, seems saner to me to just check when we're already checking if it should use the skill if for no other reason than its shorter code. poolcheck only needs to be there if another mob uses it at a different % (or doesn't have a hp check at all). Have an example though:

```lua
function onMobSkillCheck(target, mob, skill)
    if (mob:getHPP() > 50 and mob:getPool() == 3326) then
        return 1;
    end
    return 0;
end;
```
that example is if other NM uses it whenever, but the the specified NM uses it only below 50% hp. 3 lines. most mobs either don't need an onMobFight or have other checks the minions would need to learn to not break.